packages/tools/pie-icons-react/turbo.json

```diff
@@ -5,12 +5,13 @@
   ],
   "pipeline": {
     "clean": {
-      "cache": false,
+      "cache": true,
```

Comment:
If this is removing old artifacts then I'm not sure we want to ever cache this step

Reply:
I also just noticed that the icons build doesn't depend on cleaning the icons package so I'm updating that now.